README.md

```diff
@@ -1,8 +1,6 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
 # Portforio
```

Comment:
[OPTIONAL]
- [ ] Please correct this typo in your README file, I believe it's `Portfolio` not `Portforio` 🙏